packages/office-ui-fabric-react/src/components/MessageBar/docs/MessageBarDonts.md

```diff
@@ -1,2 +1,3 @@
 - Don’t use paragraphs, long sentences, or special formatting in a MessageBar. The control tries to grow to accommodate all the text and will just result in pushing the user’s main content too low on the view.
 - Don’t use buttons when a subtler link will suffice. Reserve the usage of button for when the MessageBar has a single ”hero” action that has vital usefulness to the user at that particular moment. Using more than one button is discouraged.
+- Don't overuse message bars, or load a page with multiple message bars because each MessageBar is read out loud by screen readers.
```

Comment:
Good addition! Might be good to be even more explicit that they're read on page load (assuming that's correct).
```suggestion
- Don't overuse MessageBars or load a page with multiple MessageBars, because screen readers will read each MessageBar on page load.
```